src/pages/community/how-to-contribute.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Contributing a component
+description:
+  We want anyone to be able to add assets to the community component index.
+  Maintainers can submit a short pull request in GitHub to add components to the
+  index.
+
+tabs: ['Component index', 'How to contribute']
+---
+
+<PageDescription>
+
+We want anyone to be able to add assets to the community component index.
+Maintainers can submit a short pull request in GitHub to add components to the
+index.
+
+</PageDescription>
+
+## Add a component using the GitHub UI
+
+1. In the
+   [index folder](https://github.com/carbon-design-system/carbon-website/tree/master/src/data/index)
+   of the carbon-website GitHub repo, open your team folder (or create one if
+   your team doesn’t have on yet).
+2. On the next page, in the top right above the the table click `Add file` then
+   `Create new file`.
+3. Name your file `component-name.yml`.
+4. Copy and paste this template into the “Edit new file” section of the page and
+   fill out the correct information for your component.
+   ```
+   name: component name
+   website_url: >-
+     url of documentation / guidance page (if any)
+   code_url: >-
+     url of component code on GitHub
+   maintainer: name of maintainer team
+   description: >-
+     short description of purpose and usage of component
+   framework: choose one of Vanilla | Angular | React | Vue
+   design_asset: choose one of Sketch | Axure | Adobe XD | Figma
+   platform: Web
+   aliases:
+     - fuzzy term for component (e.g. an alias for a "card" component might be "tile")
+   date_added: YYY-MM-DDT00:00:00.000Z
+   ```
+5. Once done, click the green `Create new file` button at the bottom.
```

Comment:
This button says **Commit new file** for me. Is it different for me, or is this a mistake?